migrations/migrations/0001_archive_meetings.py

```diff
@@ -0,0 +1,113 @@
+from typing import List, Optional, Set
```

Comment:
TODO: Split migration

Reply:
While doing that, I would be in favor to start creating base classes for easy migrations (in this case, adding a field with a default value, namely `limit_of_meetings`). This case will probably occur often and it would be nice to just have an `AddFieldMigration` from which you can inherit and define the attribute for the collection field and the default value.